tutorials/io/30_reading_fnirs_data.py

```diff
@@ -61,16 +62,146 @@
+# Finally, the data can be converted in to an MNE data structure.
+# The metadata above is used to create an :class:`mne.Info` data structure,
+# and this is combined with the data to create an MNE :class:`~mne.io.Raw`
+# object. For more details on the info structure see :ref:`tut-info-class`, and
+# for additional details on how continuous data is stored in MNE see
+# :ref:`tut-raw-class`.
+# For a more extensive description of how to create MNE data structures from
+# raw array data see :ref:`tut_creating_data_structures`.
+
+info = mne.create_info(ch_names=ch_names, ch_types=ch_types, sfreq=sfreq)
+raw = mne.io.RawArray(data, info, verbose=True)
+
+
+###############################################################################
+# Applying standard sensor locations to imported data
+# ---------------------------------------------------
+#
+# Having information about optode locations may assist in your analysis.
+# Beyond the general benefits this provides (e.g. creating regions of interest,
+# etc), this is may be particularly important for fNIRS as information about
+# the optode locations is required to convert the optical density data in to an
+# estimate of the haemoglobin concentrations.
+# MNE provides methods to load standard sensor configurations (montages) from
+# some vendors, and this is demonstrated below.
+# Some handy tutorials for understanding sensor locations, coordinate systems,
+# and how to store and view this information in MNE are:
+# :ref:`tut-sensor-locations`, :ref:`plot_source_alignment`, and
+# :ref:`ex-eeg-on-scalp`.
+#
+# Below is an example of how to load the optode positions for an Artinis
+# OctaMon device.
+
+montage = mne.channels.make_standard_montage('artinis-octamon')
+raw.set_montage(montage)
+
+# View the position of optodes in 2D to confirm the positions are correct.
+raw.plot_sensors()
+
+
+###############################################################################
+# To validate the positions were loaded correctly it is also possible to view
+# the location of the sources (red), detectors (black), and channels (white
+# lines and orange dots) in a 3D representation.
+# The ficiduals are marked in blue, green and red.
+# See :ref:`plot_source_alignment` for more details.
+
+subjects_dir = mne.datasets.sample.data_path() + '/subjects'
+mne.datasets.fetch_fsaverage(subjects_dir=subjects_dir)
+
+trans = mne.channels.compute_native_head_t(montage)
+
+fig = mne.viz.create_3d_figure(size=(800, 600), bgcolor='white')
+fig = mne.viz.plot_alignment(
+    raw.info, trans=trans, subject='fsaverage', subjects_dir=subjects_dir,
+    surfaces=['brain', 'head'], coord_frame='mri', dig=True, show_axes=True,
+    fnirs=['channels', 'pairs', 'sources', 'detectors'], fig=fig)
+mne.viz.set_3d_view(figure=fig, azimuth=90, elevation=90, distance=0.5,
+                    focalpoint=(0., -0.01, 0.02))
+
+
+###############################################################################
+# Storing of optode locations
```

Comment:
this whole section seems almost like a "note" that applies to all the other sections.  Maybe this info belongs at the very top of the tutorial, before the NIRx section?